portality/tasks/anon_export.py

```diff
@@ -138,7 +138,7 @@ def run_anon_export(tmpStore, mainStore, container, clean=False, limit=None, bat
         # Use the model's dump method to write out this type to file
         out_rollover_fn = functools.partial(_copy_on_complete, logger_fn=logger_fn, tmpStore=tmpStore, mainStore=mainStore, container=container)
         _ = model.dump(q=iter_q, limit=limit, transform=transform, out_template=output_file, out_batch_sizes=batch_size,
-                       out_rollover_callback=out_rollover_fn, es_bulk_fields=["_id"], scroll_keepalive='3m')
+                       out_rollover_callback=out_rollover_fn, es_bulk_fields=["_id"], scroll_keepalive='5m')
```

Comment:
not too artificially make work, but perhaps we should just put this keepalive in the config for the type, since we're making changes?